bpf/lib/policy.h

```diff
@@ -11,6 +11,7 @@
 DECLARE_CONFIG(bool, allow_icmp_frag_needed,
 	       "Allow ICMP_FRAG_NEEDED messages when applying Network Policy")
 DECLARE_CONFIG(bool, enable_icmp_rule, "Apply Network Policy for ICMP packets")
+DECLARE_CONFIG(bool, policy_accounting, "Enable policy accounting")
```

Comment:
What is policy accounting, in a nutshell? `Enable policy accounting` can be inferred from the variable name, but it would be great if we could add a few more words describing features going forward.

Also, following the enable_ convention:
```suggestion
DECLARE_CONFIG(bool, enable_policy_accounting, "Enable policy accounting")
```

Reply:
thanks, will apply